crates/sui-benchmark/src/bin/shared.rs

```diff
@@ -236,7 +237,7 @@ async fn main() {
                             make_counter_increment_transaction(gas.1 .0, package_ref, counter_id);
                         let res = qd
                             .execute_transaction(ExecuteTransactionRequest {
-                                transaction: tx,
+                                transaction: tx.clone(),
```

Comment:
Is this necessary?

Reply:
yes, because we are moving the tx object later on